docs/ai-gateway/README.md

```
@@ -0,0 +1,67 @@
+
+## Default Ports
+
+| Port | Service | Description |
+|------|---------|-------------|
+| 8080 | Router | HTTP traffic |
+| 8443 | Router | HTTPS traffic |
+| 9090 | Gateway-Controller | REST API |
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)